ctapipe/core/tool.py

```diff
@@ -173,6 +173,13 @@ def main():
         help="Logging Level for File Logging",
     ).tag(config=True)
 
+    log_include_provenance = Bool(
```

Comment:
Don't add an option for this (it's somewhat confusing to have control options affect only debugging output).  Better to just decrease the log level of that print out (could set it to 5, which is below debug), or as I mentioned below, simply remove the printout altogether.  I think I prefer the latter option to just drop it, as we already print it in the provenance file.

Reply:
We could also add use a specific provenance sublogger so it's configurable on its own.

```
self.log.getChild("provenance").debug(...)
```